### DISCUSSION TITLE
3D housing

### DISCUSSION THREAD

**Discussion by JoaquinSainz** [probonopd/MiniDexed#138] [Q&A]
Hello people, is there anyone who has 3d printing case files that they want to share or is willing to sell them?
Thanks to all the contributors of this great project

**Comment by probonopd** [comment]
Still working on mine. The fact that MiniDexed runs on all Raspberry Pi models and can use different DACs doesn't make the job easier ;)

**Reply by probonopd** [reply]
Making progress... all snap-fit, no tools needed!

![case](https://user-images.githubusercontent.com/2480569/164977033-eaa56de2-ba68-41c8-86dd-10ce0d7dc18f.jpg)

This one is for the Raspberry Pi Zero (2).
